### PR TITLE
[Simd] Add missing header files

### LINKED ISSUE #28370: [Simd] Missing header files
Only `SimdLib.hpp`, missing a lot of header files.
https://github.com/ermig1979/Simd/blob/d0f9ed1e0a179d8dd61748f61103d63c234fcd41/src/Simd/SimdLib.hpp#L28
```
#include "Simd/SimdView.hpp"
#include "Simd/SimdPixel.hpp"
#include "Simd/SimdPyramid.hpp"
```

### PR DESCRIPTION
**Pull request by JonLiu1993**
**Describe the pull request**

- #### What does your PR fix?
  Fixes #28370

When install simd, only SimdLib.hpp, missing a lot of header files. I add the missing header files.

### CODE REVIEW

File: ports/simd/portfile.cmake [hunk 1]
```diff
@@ -52,6 +52,9 @@ if(VCPKG_TARGET_IS_WINDOWS AND (VCPKG_TARGET_ARCHITECTURE STREQUAL "x86" OR VCPK
   )
   
   file(COPY "${SOURCE_PATH}/src/Simd/SimdLib.hpp" DESTINATION "${CURRENT_PACKAGES_DIR}/include")
+  file(COPY "${SOURCE_PATH}/src/Simd/SimdView.hpp" DESTINATION "${CURRENT_PACKAGES_DIR}/include")
+  file(COPY "${SOURCE_PATH}/src/Simd/SimdPixel.hpp" DESTINATION "${CURRENT_PACKAGES_DIR}/include")
+  file(COPY "${SOURCE_PATH}/src/Simd/SimdPyramid.hpp" DESTINATION "${CURRENT_PACKAGES_DIR}/include")
```
Comment (Thomas1664) [inline ports/simd/portfile.cmake:57]: I would use a GLOB here to make it more maintainable if upstream adds more headers in the future.

File: versions/s-/simd.json [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+    {
+      "git-tree": "b3bda46ed7952e5766bafeab1f94d6f03bd62107",
+      "version": "5.1.119",
+      "port-version": 1
+    },
     {
       "git-tree": "56af2de7152865b9e3ede88a7604dd54ba4251d9",
-      "version": "5.1.119"
+      "version": "5.1.119",
+      "port-version": 0
```
Comment (Thomas1664) [inline versions/s-/simd.json:11]: Version database compromised

**Comment by Thomas1664** [comment]
Furthermore, this port has a CMake build system: https://github.com/ermig1979/Simd/blob/96dfde3b243687d796426719cf354b47fd4cd727/prj/cmake/CMakeLists.txt

File: versions/s-/simd.json (hunk 2 above)
Comment (JonLiu1993) [reply versions/s-/simd.json:11]: Here is automatically added after using the command "./vcpkg x-add-version simd --overwrite-version ", we should keep Just add "port-version": 0 in here,

**Comment by JonLiu1993** [comment]
> Furthermore, this port has a CMake build system: https://github.com/ermig1979/Simd/blob/96dfde3b243687d796426719cf354b47fd4cd727/prj/cmake/CMakeLists.txt

We will not consider replacing the cmake build system for this PR, maybe the next version update will consider doing so.

**Review by Adela0814 — APPROVED** [review]

File: versions/s-/simd.json (hunk 2 above)
Comment (vicroms) [reply versions/s-/simd.json:11]: It appears to me that the original entry was written manually and thus lacked the "port-version" line. This should be fine as they are semantically the same (same `git-tree`).

**Pull request merged**